app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -652,9 +676,9 @@ function AcuantCapture(
       {isCapturingEnvironment && selfieCapture && (
         <AcuantSelfieCamera
           onImageCaptureSuccess={onSelfieCaptureSuccess}
-          onImageCaptureFailure={onSelfieCaptureFailure}
-          onImageCaptureOpen={() => setIsCapturingEnvironment(true)}
-          onImageCaptureClose={() => setIsCapturingEnvironment(false)}
+          onImageCaptureFailure={(error) => onSelfieCaptureFailure(error)}
```

Comment:
This can be changed to 

`onImageCaptureFailure={onSelfieCaptureFailure}`

with change in the _acuant-selfie-camera.tsx_

```typescript
/**
   * Failure callback
   */
  onImageCaptureFailure: ({ error }: { error: any }) => void;
```

Reply:
@night-jellyfish , i think what you mentioned makes sense, it should be following? error has a type of hash/object, with key of code and message. The method takes one parameter of the defined hash/object.

`  onImageCaptureFailure: ( error : { code: integer, message: string }) => void;`

Reply:
Thank you! I've just pushed that up in [64d9c84](https://github.com/18F/identity-idp/pull/9795/commits/64d9c8400ffd4f900a0f9f578f996a9ed14f8826).